src/lib/badge/badge.html

```diff
@@ -0,0 +1,5 @@
+<ng-content></ng-content>
+<span 
+  class="mat-badge-content" 
+  [innerHTML]="content">
```

Comment:
This looks dangerous. Is there any reason people would want to put anything besides plain text in a badge?

Reply:
Removed in latest.